e2e_samples/temperature_events/LoadTesting/IoTSimulator.ps1

```diff
@@ -0,0 +1,86 @@
+<#
+This script creates several Azure Container Instances that runs the IoT Simulator.
+All resources, including the resource group, will be torn down automatically 
+after the simulators finish sending the load.
+#>
+param (
+    [string]$Location = 'japaneast',
+    [Parameter(Mandatory=$true)][string]$ResourceGroup,
+    [int]$DeviceCount = 1,
+    [int]$ContainerCount = 1,
+    [int]$MessageCount = 1,
+    [int]$Interval = 1000,
+    [string]$FixPayload = '',
+    [int]$FixPayloadSize = 0,
+    [string]$Template = '{ \"deviceId\": \"$.DeviceId\", \"temp\": $.Temp, \"Ticks\": $.Ticks, \"Counter\": $.Counter, \"time\": \"$.Time\", \"engine\": \"$.Engine\", \"source\": \"$.MachineName\" }',
+    [string]$Header = '',
+    [string]$Variables = '[{name: \"Temp\", random: true, max: 25, min: 23}, {name:\"Counter\", min:100}, {name:\"Engine\", values: [\"on\", \"off\"]}]',
+    [Parameter(Mandatory=$true)][string]$EventHubConnectionString,
+    [string]$Image = "iottelemetrysimulator/azureiot-telemetrysimulator:latest",
+    [double]$Cpu = 1.0,
+    [double]$Memory = 1.5
+ )
+
+<#
+We want the load to generate:
+- 50% of the load where deviceId > 1,000. The system is not interested in this data so these should be filter it out.
+- 40% "good" data where deviceId <1,000 AND temperature <100
+- 10% "bad" data where deviceId <1,000 AND temperature >100
+#>
+$FilteredTemplate = '{ \"deviceId\": \"$.FilteredDeviceId\", \"temp\": $.Temp, \"time\": \"$.Time\" }' 
+$GoodTemplate = '{ \"deviceId\": \"$.DeviceId\", \"temp\": $.Temp, \"time\": \"$.Time\" }'
+$BadTemplate = '{ \"deviceId\": \"$.DeviceId\", \"temp\": $.BadTemp, \"time\": \"$.Time\" }' 
+$Variables = '[{name: \"DeviceId\", random: true, max: 1000, min: 0}, {name: \"FilteredDeviceId\", random: true, min: 1000}, {name: \"Temp\", random: true, max: 100, min: 0}, {name: \"BadTemp\", random: true, min: 100}]'
```

Comment:
Assuming integers: `...{name: \"Temp\", random: true, max: 99, min: 0}, {name: \"BadTemp\", random: true, min: 100}`

Reply:
OK I updated the linked issue with fixed requirements

Reply:
good catch haha let me fix it, thanks :)

Reply:
fixed, thanks :)